addons/event/models/event_question.py

```diff
@@ -76,9 +76,12 @@ def _unlink_except_default_question(self):
 
     def action_view_question_answers(self):
         """ Allow analyzing the attendees answers to event questions in a convenient way:
+
         - A graph view showing counts of each suggestion for simple_choice questions
           (Along with secondary pivot and list views)
-        - A list view showing textual answers values for text_box questions. """
+
```

Comment:
Only the empty new line at line 79 is necessary. The empty new line at line 82 forces rst to create another list.